File: system/testSoluttions/battleshipGame/template.java
import java.util.Scanner;

public class solution2{
    static Scanner scanner = new Scanner(System.in);
    public static void main(String[] args) {
        //Запишите сюда первый ход в формате "строка-столбец"
        String firstAction = "";
        System.out.println(firstAction);
        System.out.flush();
        int row;
        int column;
        while (true) {
            //Ответ на выстрел
            int answer = Integer.parseInt(scanner.nextLine());
            //Заполните переменные row и column

            String response = row + "-" + column;
            System.out.println(response);
            System.out.flush();
        }
    }
}

File: system/testSoluttions/battleshipGame/template.php
<?php
$answer = 0;
$response = "";
#Запишите сюда первый ход в формате "строка-столбец"
$firstAction = '';
echo("{$firstAction}\r\n");
$row = -1;
$column = -1;

while (TRUE) {
    #Ответ на выстрел
    $answer = (integer)readline();
    #Заполните переменные row и column

    $response = "{$row}-{$column}";
    echo("{$response}\r\n");
}
?>